modules/hpe_nimble_acr.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/python
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# author alok ranjan (alok.ranjan2@hpe.com)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+author:
+    - Alok Ranjan (@ranjanal)
+description: On HPE Nimble Storage Array - Create or delete ACR for volume.
+module: hpe_nimble_acr
+
+options:
+    state:
```

Comment:
As a usual recommendation.. all python codes follows 4 space indentation.  Is this a problem because rendering is happening correctly. ? The problem is if i change my visual studio settings to follow 2 space for "TAB" then while coding it will create problem. I can only have one setting in editor. 

https://www.python.org/dev/peps/pep-0008/#:~:text=The%204%2Dspace%20rule%20is%20optional%20for%20continuation%20lines.

[4 space stackoverflow](https://stackoverflow.com/questions/1125653/python-using-4-spaces-for-indentation-why)

Reply:
moved to 2 space for yaml